### DISCUSSION TITLE
Planning

### DISCUSSION THREAD

**Discussion by mamamia5x** [mamamia5x/markdownpedia#6] [General]
Plans for Markdownpedia

* Editor
  * Allow users to edit on the site.
* Add links to Readme
* Credits page
  * Give credits to users who've helped make this site.

**Comment by riungemaina** [comment]
What's the focus? of the content? is it going to be general stuff like Wikipedia? or dev focused? programming?

**Reply by mamamia5x** [reply]
Good question. Currently, it's kinda both. By that, I mean I want the pages done, and the programming. Currently, I personally am working on programming, because I would like the site to look natural and better. What I want is for other people to make and edit pages, but I can't control that and it is up to the people to do as they wish. So honestly, it's up to you. I'm just happy that other people are working on the project, and together we're making the site look great.

**Reply by riungemaina** [reply]
Okay, if I find some time, I'll find something I can help with.

**Reply by mamamia5x** [reply]
Alright, thanks.
